bugbug/similarity.py

```diff
@@ -106,6 +111,21 @@ def get_text(self, bug, all_comments=False):
 
         return "{} {}".format(bug["summary"], comments)
 
+    def get_similar_bugs(self, bug):
+        similar_bug_ids = self.get_similar_bugs_child(bug)
+        similar_bugs = bugzilla.get(similar_bug_ids)
+        if self.end_to_end:
+            duplicatemodel = DuplicateModel.load("duplicatemodel")
+            bug_couples = [(bug, similar_bugs[bug_id]) for bug_id in similar_bugs]
+            probs = duplicatemodel.classify(bug_couples, probabilities=True)
+            similar_bug_ids = [
+                similar_bug_ids[idx]
+                for idx, prob in enumerate(probs)
+                if prob[1] > self.confidence_threshold
+            ]
```

Comment:
As a side note, maybe we should also sort them by prob (higher prob should come first).